src/main/java/uk/gov/hmcts/reform/ethos/ecm/consumer/helpers/transfertoecm/TransferToEcmCaseDataHelper.java

```diff
@@ -37,11 +39,10 @@
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.CLOSED_STATE;
 import static uk.gov.hmcts.ecm.common.model.helper.Constants.POSITION_TYPE_CASE_TRANSFERRED_OTHER_COUNTRY;
 
+@Slf4j
+@Component
```

Comment:
Why is this now a component?

Reply:
I have added one non-static method to work around the difficulty of mocking static method for the unit test that checks office name in the case transfer.